app/javascript/components/types.ts

```diff
@@ -441,6 +441,7 @@ export type Representation = {
   track: { title: string; iconUrl: string }
   numSubmissions: number
   feedbackHtml: string
+  feedbackType:RepresentationFeedbackType|""
```

Comment:
I'll fix this to return `null` if it isn't already set. Makes more sense

Reply:
ok, yes I have derived this from the current return format